diff --git a/external/llvh/lib/Support/CMakeLists.txt b/external/llvh/lib/Support/CMakeLists.txt
index f1ccbf0..213cdc2 100644
--- a/external/llvh/lib/Support/CMakeLists.txt
+++ b/external/llvh/lib/Support/CMakeLists.txt
@@ -1,6 +1,7 @@
 add_hermes_library(LLVHSupport
     APFloat.cpp
     APInt.cpp
+    Atomic.cpp
     circular_raw_ostream.cpp
     CommandLine.cpp
     ConvertUTF.cpp
-- 
2.49.0

