diff --git a/external/llvh/lib/Support/Windows/Threading.inc b/external/llvh/lib/Support/Windows/Threading.inc
index 6913297..53429de 100644
--- a/external/llvh/lib/Support/Windows/Threading.inc
+++ b/external/llvh/lib/Support/Windows/Threading.inc
@@ -14,7 +14,7 @@
 #include "llvh/ADT/SmallString.h"
 #include "llvh/ADT/Twine.h"
 
-#include "Windows/WindowsSupport.h"
+#include "WindowsSupport.h"
 #include <process.h>
 
 // Windows will at times define MemoryFence.
-- 
2.49.0

